esx_service/utils/kvESX.py

```diff
@@ -256,32 +257,42 @@ def load(volpath):
     """
     Load and return dictionary from the sidecar
     """
+    logging.info("Load dictionary from sidecar for vol path %s", volpath)
```

Comment:
it feels like a debug, not info. Can you elaborate why do we want it on every log all the time ?